code-challenges.test.js

```diff
@@ -11,40 +11,259 @@
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+// Snapshots:   0 total
+// Time:        0.442 s
+// Ran all test suites.
+// ✨  Done in 1.31s.
+
+
+
+// Question2) 
+
+// Create a function that takes in an object that contains up votes and down votes and returns the net total of votes.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const votes1 = {upVotes: 13, downVotes: 2}
+// const votes1 = {upVotes: 13, downVotes: 2}
 // Expected output: 11
-const votes2 = {upVotes: 2, downVotes: 33}
+// const votes2 = {upVotes: 2, downVotes: 33}
 // Expected output: -31
 
+// a) Create a test with expect statements for each of the variables provided.
+
+describe("mostPopularVotes", () => {
+    it("takes in an object that contains up votes and down votes and returns the net total of votes", () => {
+        expect(mostPopularVotes(votes1)).toEqual(11)
+        expect(mostPopularVotes2(votes2)).toEqual(-31)
+    })
+})
+
+
+// FAIL  ./code-challenges.test.js
+// mostPopularVotes
+//   ✕ takes in an object that contains up votes and down votes and returns the net total of votes (1 ms)
+
+// ● mostPopularVotes › takes in an object that contains up votes and down votes and returns the net total of votes
+
+//   ReferenceError: mostPopularVotes is not defined
+
+//     102 | describe("mostPopularVotes", () => {
+//     103 |     it("takes in an object that contains up votes and down votes and returns the net total of votes", () => {
+//   > 104 |         expect(mostPopularVotes(votes1)).toEqual(11)
+//         |         ^
+//     105 |         expect(mostPopularVotes(votes2)).toEqual(-31)
+//     106 |     })
+//     107 | })
+
+//     at Object.expect (code-challenges.test.js:104:9)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
+// Time:        0.431 s, estimated 1 s
+
 
 // b) Create the function that makes the test pass.
 
+// Create a function that takes in an object that contains up votes and down votes and returns the net total of votes.
 
-// --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
+// INPUT: Function that takes in n object that contains up votes and down votes 
+// OUTPUT: returns the net total of votes.(11,-31)
+// CONCLUSION: Create variables for up votes and down votes minus the up votes and the down votes thrn return sum of the votes 
+
+// const votes1 = {upVotes: 13, downVotes: 2}
+// Expected output: 11
+// const votes2 = {upVotes: 2, downVotes: 33}
+// Expected output: -31
+
+
+const upVotes = 13
+const downVotes= 2
+const votes1 = upVotes - downVotes
+var mostPopularVotes = () => {
+    return votes1
+
+}
+console.log(mostPopularVotes())
+
+
+const upVotes2 = 2
+const downVotes2= 33
+const votes2 = upVotes2 - downVotes2
+var mostPopularVotes2 = () => {
+    return votes2
+
+}
```

Comment:
I like where your head is at but remeber these value are just test values. You are not going ot know what votes your users are giving you as a dev so you have to make code that works for all votes and considers the votes objects.